pkg/kademlia/config.go

```diff
@@ -54,30 +54,17 @@ func (c Config) Verify(log *zap.Logger) error {
 
 // OperatorConfig defines properties related to storage node operator metadata
 type OperatorConfig struct {
```

Comment:
The email must still be specified for answering info query for trusted satellite. Also removing this is the cause for build failures.

Reply:
Then we can't remove emails from Kademlia until that is fixed.